### ISSUE TITLE
My antivirus said it has a virus.

### ISSUE THREAD

**Issue by Orangeguyosc** [olegos2/mobox#297]
I downloaded one of the apk and my antivirus said it has a virus called "evo-gen" does your apk file actually has a virus?

**Comment by Negrobear** [comment]
Whaaaaaaaaat?

**Comment by Jacobw1oo** [comment]
yes, input bridge is know, to cause anti virus to think it a virus. it because of how Input bridge acts. but Input bridge is not open source and might be a virus, it seems to use an insane about of CPU.
